src/pages/components/NewComment.jsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import InputGroup from 'react-bootstrap/InputGroup';
+import axios from 'axios';
+
+const NewComment = ({ article_id, baseURL, setCommentList }) => {
+  const [username, setUsername] = useState('');
+  const [commentBody, setCommentBody] = useState('');
+  const [isSubmitting, setSubmitting] = useState(false);
+  const [isError, setError] = useState(false);
+
+  const handleNewComment = async (e) => {
+    e.preventDefault();
+    setSubmitting(true);
+    setError(false);
+    try {
+      const response = await axios.post(
+        `${baseURL}articles/${article_id}/comments`,
+        {
+          author: username || 'Anon',
+          body: commentBody,
+        }
+      );
+      setCommentList((prevComments) => [
+        ...prevComments,
+        response.data.comment,
+      ]);
+      setUsername('');
+      setCommentBody('');
+    } catch (err) {
+      setError(true);
+      throw new Error('Error posting new comment: ', err);
+    } finally {
+      setSubmitting(false);
+    }
+  };
+
+  return (
+    <Form onSubmit={handleNewComment}>
+      <InputGroup className="mb-3">
+        <InputGroup.Text id="basic-addon1">@</InputGroup.Text>
+        <Form.Control
+          value={username}
+          onChange={(e) => setUsername(e.target.value)}
+          placeholder="Username"
+          aria-label="Username"
+        />
+      </InputGroup>
```

Comment:
This should be hard coded for now instead of given as an option. You can assume the user is already logged in for an easier user experience. As I actually need to know a valid username to be able to post otherwise